Jenkinsfile

```diff
@@ -100,6 +100,10 @@ def init_git() {
     script: './tests/scripts/task_show_node_info.sh',
     label: 'Show executor node info',
   )
+  sh (
```

Comment:
i think this might mean that different parts of the build use different merge commits, depending on when this runs, right?

Reply:
fixed in the update, it fetches origin once at the beginning and checks out that same commit in further stages